subsys/net/ip/Kconfig.rpl

```diff
@@ -232,7 +232,6 @@ config	NET_RPL_PREFERENCE
 config	NET_RPL_GROUNDED
 	bool "DAG grounded default value"
 	depends on NET_RPL
-	default false
```

Comment:
It looks this change goes with the previous commit's message, and vice versa?

Reply:
Oops... git slip-up. Fixed.